cockroachcloud/run-bulk-operations.md

```diff
@@ -5,6 +5,8 @@ toc: true
 docs_area: manage
 ---
 
+{% include cockroachcloud/ccloud/backup-types.md %}
+
```

Comment:
And since the Run Bulk Operations page is showing users how to run their own backups, can you add a sentence to that effect directly on the page after the include reference? 

> The examples on this page demonstrate how to take customer-owned backups.